### ISSUE TITLE
Humming Noise with constant slider use

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/molarity#202]
**Test device**
Dell
**Operating System**
Win 10
**Browser**
Edge (and other browsers)
**Problem description**
For https://github.com/phetsims/QA/issues/464.
If you hold down shift+arrow to move the sliders, the dings become a hum that gets louder and does not seem connected to the intended sounds.

**Visuals**
https://drive.google.com/file/d/1r5VLv8bQYLhU1owIXYFC6lphmoiy6xxA/view?usp=sharing

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Molarity‬
URL: https://phet-dev.colorado.edu/html/molarity/1.5.0-rc.1/phet/molarity_all_phet.html
Version: 1.5.0-rc.1 2019-12-19 01:13:41 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.102 Safari/537.36 Edge/18.18363
Language: en-US
Window: 1536x751
Pixel Ratio: 2.5/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0
Vendor: Microsoft (Microsoft Edge)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by twant** [comment]
I don't know enough about the sonification design & implementation to know exactly how to address this (or what the ideal sound would be for this situation). @jbphet and @emily-phet, do you mind taking a look at this one to see what you think of the current sound situation when holding down shift + arrow key on one of the sliders?

**Comment by jbphet** [comment]
@KatieWoe said:

> [T]he dings become a hum that gets louder and does not seem connected to the intended sounds.

The "hum" is just rapid sequential playing of the individual sounds, each of which indicates the concentration level.  I suspect that it appears to get louder only because it starts low and goes higher, and the speakers that were used for the test don't have much bass response.

It would be possible to prevent this particular behavior by adding a minimum change amount to the `ConcentrationSoundGenerator` before playing a sound.  However, I'm not sure that we want to do this.  @terracoda has told me that people rarely use the shift-arrow capability, and holding shift-arrow is even more rare.  So, by adding a minimum threshold, we'd be trading off the ability to play a sound on every key event for prevention of the continuous-seeming tone in a rare use case.  This doesn't seem worth it to me.

@terracoda - In your opinion, is it better to have a sound produced on every keyboard action or to add a threshold to prevent continuous sounds when shift-up-arrow or shift-down-arrow is held by a user?  Please assign back to me once you've logged your input.

**Comment by terracoda** [comment]
I agree with @jbphet, holding the shift key + an Arrow key is very rare if not non-existant. The sound design is not meant to encourage this kind of interaction.

What I think the sound design does nicely here is two things:
1. marks meaningful changes with regular increments (no shift key)
2. With shift key marks tiny changes in concentration - hardly significant changes.

I would support not changing the current design.

**Comment by zepumph** [comment]
From this I'm hearing that we are not going to change this. Tagging as "wont fix" and closing.
